department-of-veterans-affairs/va.gov-cms
# VACMS-2455 media list images content type

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2455

## Testing done


## Screenshots


## QA steps

As an admin
1. Login
1. Navigate here: admin/structure/types/manage/media_list_images
3. Confirm the following AC
 - [x] content type "Media list - Images" | media_list_images exists
    - [x] add field "Alert" field_alert | entity revisions reference to "Alert (Single)" paragraph only | cardinality single | 
           optional directly under "Title and introduction" fieldset
![image](https://user-images.githubusercontent.com/5752113/91500382-69229f00-e891-11ea-95ef-283339f6e41e.png)


    - [x] add field_media_list_images below CTA buttons field | label "Images" | entity reference to paragraph type "Media 
           list - Images" | cardinality - single

![image](https://user-images.githubusercontent.com/5752113/91500425-82c3e680-e891-11ea-88e9-3295acebecd8.png)


    - [x] editorial / node view experience is using established patterns and does not have UX regressions
![image](https://user-images.githubusercontent.com/5752113/91500467-93745c80-e891-11ea-9a61-62ba2984b81e.png)


    - [x] spec tool / tests update
4. Regression test the Alert Single component field on the Media List - Images content type, the Media List - Videos content type as well as the Step-by-step content type.
  - [x] Alert single behaviors still work.

## Definition of Done
